volatility3/framework/plugins/linux/mountinfo.py

```diff
@@ -36,7 +36,7 @@ class MountInfo(plugins.PluginInterface):
     """Lists mount points on processes mount namespaces"""
 
     _required_framework_version = (2, 2, 0)
-    _version = (1, 2, 3)
```

Comment:
I don't think this is an externally visible change (the inputs stay the same and I expect the outputs to be similar too), so should be fine to leave it as it is, but also happy to have it bumped.

Reply:
done, updated only the patch version